src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/flyout/flyout.js

```diff
@@ -324,6 +324,7 @@ export class Flyout extends Component {
           const options = this.state.indexPatterns.map(indexPattern => ({
             text: indexPattern.get('title'),
             value: indexPattern.id,
+            ['data-test-subj']: `indexPatternOption${indexPattern.get('title')}`,
           }));
```

Comment:
Let's add a `-` here so we have `indexPatternOption-${indexPattern.get('title')}`

Reply:
added